inbm/dispatcher-agent/dispatcher/sota/downloader.py

```diff
@@ -88,7 +88,10 @@ def download(self,
                      "perform a software update")
 
     def check_release_date(self, release_date: Optional[str]) -> bool:
-        pass
+        """
+        @return True alwyas as ubuntu doesn't need to check release date
```

Comment:
always is spelled wrong in the comment.

Reply:
We will correct this typo mistake in next PR.